libmscore/iname.h

```diff
@@ -30,6 +31,7 @@ class InstrumentName final : public TextBase
 {
     InstrumentNameType _instrumentNameType;
     int _layoutPos { 0 };
+    SysStaff* _sysStaff { 0 };
```

Comment:
```c++
SysStaff* _sysStaff { nullptr };
```
Note: `_layoutPos` in the line above is an `int` not a pointer so there is no inconsistency.